topics/single-cell/tutorials/GO-enrichment/data-library.yaml

```diff
@@ -0,0 +1,35 @@
+---
+destination:
+  type: library
+  name: GTN - Material
+  description: Galaxy Training Network Material
+  synopsis: Galaxy Training Network Material. See https://training.galaxyproject.org
+items:
+- name: New topic
+  description: Topic summary
+  items:
+  - name: GO Enrichment Analysis on Single-Cell RNA-Seq Data
+    items:
+    - name: 'DOI: 10.5281/zenodo.13461890'
+      description: latest
+      items:
+      - url: https://zenodo.org/api/records/13461890/files/Galaxy3-[GO].obo/content
+        src: url
+        ext: ''
+        info: https://zenodo.org/uploads/13461890
```

Comment:
those info links are not working.

Reply:
@bgruening I changed the links, hopefully they work now